tool/jt.rb

```diff
@@ -1134,7 +1134,8 @@ def test_specs(command, *args)
 
       options += %w[--excl-tag graalvm]
       options << '-t' << ENV['AOT_BIN']
-      options << '-T-XX:OldGenerationSize=2G'
+      options << '-T-XX:YoungGenerationSize=2G'
+      options << '-T-XX:OldGenerationSize=4G'
```

Comment:
I can try that. Previously we were using 256 MB / 2 GB. Bumping them did increase collection time, but there were much fewer collections. With 2GB / 4GB, I see:

```
[Incremental GC (CollectOnAllocation.Sometimes) 2089337K->113907K, 0.3720967 secs]
[Full GC (CollectOnAllocation.Sometimes) 2203267K->102357K, 0.2234189 secs]
[Full GC (CollectOnAllocation.Sometimes) 2192367K->110995K, 0.2211100 secs] 
```